README.md

```diff
@@ -1,49 +1,53 @@
-# Alert Logic Microsoft Azure Event Hub Events Collector
+# Alert Logic Event Collector for Microsoft Azure Event Hubs
 
 [![Build Status](https://secure.travis-ci.org/alertlogic/ehub-collector.png?branch=master)](http://travis-ci.org/alertlogic/ehub-collector)
 
-AlertLogic Azure Event Hub Collector
-
 # Overview
 
-This repository contains the Microsoft Azure web application Node.js source code and an Azure Resource Manager (ARM) template to set up a data collector in Azure, which collects and forwards Event Hub events to the Alert Logic backend.
+This repository contains the Microsoft Azure web application Node.js source code and Azure Resource Manager (ARM) template required to set up a data collector in Azure to collect and forward events from Event Hubs to Alert Logic.
 
 # Installation
 
-To perform the set up required to grant Alert Logic permission access to collect Event hub events, you must have access to the following:
+To perform the setup required to grant Alert Logic permission to access Events Hub for event collection, you must have:
 
 * A Microsoft Azure account with administrative privileges
 * An Alert Logic user account with administrative privileges
 
 
-## Register a New Azure Web Application
+## Register a new Azure web application
 
-In the Azure AD portal, you must register a new web application. 
+In the Azure portal, you must register a new web application. Any application that wants to use the capabilities of Azure Active Directory must first be registered in an Azure AD tenant. This registration process involves giving Azure AD details about your application, such as the URL where it iss located, the URL to send replies after a user is authenticated, and the URI that identifies the app.
```

Comment:
Typo `iss`

Reply:
Typo corrected.

Reply:
@rhosmo could you please `git push` you latest changes?